mlocal/frags/Makefile.stub

```diff
@@ -34,16 +34,15 @@ dist:
 .PHONY: unit-test
 unit-test:
 	@echo " TEST sudo go test [unit]"
-	$(V)cd $(SOURCEDIR) && sudo -E $(GO) test $(GO_MODFLAGS) -count=1 -timeout=20m -tags "$(GO_TAGS)" -failfast -cover -race \
+	$(V)$(SOURCEDIR) && sudo -E scripts/go-test \
 		`$(GO) list ./... | grep -v "cmd/singularity\|pkg/network\|e2e"`
 	@echo "       PASS"
 
 
 .PHONY: e2e-test
 e2e-test:
 	@echo " TEST sudo go test [e2e]"
-	$(V)cd $(SOURCEDIR) && sudo -E $(GO) test $(GO_MODFLAGS) -count=1 -timeout=30m -tags "$(GO_TAGS)" -failfast -cover -race \
-		./e2e
+	$(V)$(SOURCEDIR)/scripts/e2e-test
```

Comment:
missing `cd`

Reply:
this is the source of my previous mistake: the e2e-test script will locate the root directory of the git repo (this assumes it's running with a checkout copy of the repo, and not a tarball). I'll change this so that it behaves like the other tests.